views/partials/prs.hbs

```diff
@@ -75,6 +75,6 @@
 
     <div class="br2 center mt5 overflow-hidden w-50-ns w-80-m w-90 mt2">
         <button class="u-bgc-21ffff u-c-411e2f center mt2 h2 bn pointer br1 pv0 ph3 db saveUser">This is Me</button>
-        <p class='light-silver center tc-ns'>In the future, you can find your PRs by visiting <a href='{{ hostname }}/me' class='link orange underline-hover saveUser'>{{ hostname }}/me</a> on this device.</p>
+        <p class='light-silver center tc tc-ns'>In the future, you can find your PRs by visiting <a href='{{ hostname }}/me' class='link orange underline-hover saveUser'>{{ hostname }}/me</a> on this device.</p>
```

Comment:
I think we can get rid of the `tc-ns` as that's was saying 'text center not small` but I think we're saying we want it centered now on small (mobile) and not small (desktop).

Reply:
For sure, I've removed it. I just think this looks better on mobile.